lib/optional_query_cache/version.rb

```diff
@@ -0,0 +1,11 @@
+require 'active_record'
```

Comment:
What is the stack trace when it breaks?  Also, I'm not sure `version.rb` is the right place for this to live...